From: Michael Davidsaver <mdavidsaver@bnl.gov>
Date: Mon, 26 Apr 2010 14:35:27 -0400
Subject: remove duplicate adls and opis

Keep

sscan/sscanApp/op/adl/arrayPlot.adl
optics/opticsApp/op/adl/fb_epid.adl
love/loveApp/op/adl/LoveController.adl
---
 love/loveApp/op/adl/LoveController.adl          |  425 --------
 optics/documentation/fb_epid/source/fb_epid.adl | 1194 -----------------------
 sscan/sscanApp/op/opi/arrayPlot.opi             |  684 -------------
 std/stdApp/op/adl/arrayPlot.adl                 |  145 ---
 std/stdApp/op/opi/arrayPlot.opi                 |  684 -------------
 5 files changed, 3132 deletions(-)
 delete mode 100644 love/loveApp/op/adl/LoveController.adl
 delete mode 100644 optics/documentation/fb_epid/source/fb_epid.adl
 delete mode 100644 sscan/sscanApp/op/opi/arrayPlot.opi
 delete mode 100644 std/stdApp/op/adl/arrayPlot.adl
 delete mode 100644 std/stdApp/op/opi/arrayPlot.opi

diff --git a/love/loveApp/op/adl/LoveController.adl b/love/loveApp/op/adl/LoveController.adl
deleted file mode 100644
index 1d4ae04..0000000
--- a/love/loveApp/op/adl/LoveController.adl
+++ /dev/null
@@ -1,425 +0,0 @@
-
-file {
-	name="/home/oxygen12/DKLINE/EPICS/synApps/ioc/Love/LoveApp/op/adl/LoveController.adl"
-	version=030004
-}
-display {
-	object {
-		x=251
-		y=117
-		width=184
-		height=145
-	}
-	clr=0
-	bclr=8
-	cmap=""
-	gridSpacing=5
-	gridOn=0
-	snapToGrid=0
-}
-"color map" {
-	ncolors=65
-	colors {
-		ffffff,
-		ececec,
-		dadada,
-		c8c8c8,
-		bbbbbb,
-		aeaeae,
-		9e9e9e,
-		919191,
-		858585,
-		787878,
-		696969,
-		5a5a5a,
-		464646,
-		2d2d2d,
-		000000,
-		00d800,
-		1ebb00,
-		339900,
-		2d7f00,
-		216c00,
-		fd0000,
-		de1309,
-		be190b,
-		a01207,
-		820400,
-		5893ff,
-		597ee1,
-		4b6ec7,
-		3a5eab,
-		27548d,
-		fbf34a,
-		f9da3c,
-		eeb62b,
-		e19015,
-		cd6100,
-		ffb0ff,
-		d67fe2,
-		ae4ebc,
-		8b1a96,
-		610a75,
-		a4aaff,
-		8793e2,
-		6a73c1,
-		4d52a4,
-		343386,
-		c7bb6d,
-		b79d5c,
-		a47e3c,
-		7d5627,
-		58340f,
-		99ffff,
-		73dfff,
-		4ea5f9,
-		2a63e4,
-		0a00b8,
-		ebf1b5,
-		d4db9d,
-		bbc187,
-		a6a462,
-		8b8239,
-		73ff6b,
-		52da3b,
-		3cb420,
-		289315,
-		1a7309,
-	}
-}
-text {
-	object {
-		x=4
-		y=53
-		width=90
-		height=10
-	}
-	"basic attribute" {
-		clr=0
-		fill="outline"
-	}
-	textix="Readout"
-	align="horiz. centered"
-}
-text {
-	object {
-		x=4
-		y=92
-		width=85
-		height=10
-	}
-	"basic attribute" {
-		clr=0
-		fill="outline"
-	}
-	textix="Setpoint 1"
-	align="horiz. centered"
-}
-text {
-	object {
-		x=4
-		y=132
-		width=85
-		height=10
-	}
-	"basic attribute" {
-		clr=0
-		fill="outline"
-	}
-	textix="Setpoint 2"
-	align="horiz. centered"
-}
-"text update" {
-	object {
-		x=4
-		y=21
-		width=90
-		height=30
-	}
-	monitor {
-		chan="$(P)$(Q):Value"
-		clr=20
-		bclr=14
-	}
-	clrmod="alarm"
-	align="horiz. centered"
-	limits {
-	}
-}
-"text update" {
-	object {
-		x=4
-		y=65
-		width=85
-		height=25
-	}
-	monitor {
-		chan="$(P)$(Q):SetPt1"
-		clr=60
-		bclr=14
-	}
-	clrmod="alarm"
-	align="horiz. centered"
-	limits {
-	}
-}
-"text update" {
-	object {
-		x=4
-		y=104
-		width=85
-		height=25
-	}
-	monitor {
-		chan="$(P)$(Q):SetPt2"
-		clr=60
-		bclr=14
-	}
-	clrmod="alarm"
-	align="horiz. centered"
-	limits {
-	}
-}
-text {
-	object {
-		x=94
-		y=92
-		width=85
-		height=10
-	}
-	"basic attribute" {
-		clr=0
-		fill="outline"
-	}
-	textix="Alarm High"
-	align="horiz. centered"
-}
-text {
-	object {
-		x=94
-		y=132
-		width=85
-		height=10
-	}
-	"basic attribute" {
-		clr=0
-		fill="outline"
-	}
-	textix="Alarm Low"
-	align="horiz. centered"
-}
-"text update" {
-	object {
-		x=94
-		y=65
-		width=85
-		height=25
-	}
-	monitor {
-		chan="$(P)$(Q):AlarmHi"
-		clr=60
-		bclr=14
-	}
-	clrmod="alarm"
-	align="horiz. centered"
-	limits {
-	}
-}
-"text update" {
-	object {
-		x=94
-		y=104
-		width=85
-		height=25
-	}
-	monitor {
-		chan="$(P)$(Q):AlarmLo"
-		clr=60
-		bclr=14
-	}
-	clrmod="alarm"
-	align="horiz. centered"
-	limits {
-	}
-}
-rectangle {
-	object {
-		x=99
-		y=21
-		width=80
-		height=38
-	}
-	"basic attribute" {
-		clr=14
-	}
-}
-text {
-	object {
-		x=55
-		y=3
-		width=125
-		height=15
-	}
-	"basic attribute" {
-		clr=30
-		fill="outline"
-	}
-	textix="$(P)$(Q)"
-	align="horiz. centered"
-}
-rectangle {
-	object {
-		x=110
-		y=27
-		width=10
-		height=10
-	}
-	"basic attribute" {
-		clr=10
-	}
-}
-rectangle {
-	object {
-		x=133
-		y=28
-		width=10
-		height=10
-	}
-	"basic attribute" {
-		clr=10
-	}
-}
-rectangle {
-	object {
-		x=160
-		y=27
-		width=10
-		height=10
-	}
-	"basic attribute" {
-		clr=10
-	}
-}
-menu {
-	object {
-		x=0
-		y=0
-		width=55
-		height=14
-	}
-	control {
-		chan="$(P)$(Q):Disable"
-		clr=14
-		bclr=51
-	}
-}
-text {
-	object {
-		x=104
-		y=40
-		width=20
-		height=10
-	}
-	"basic attribute" {
-		clr=0
-		fill="outline"
-	}
-	textix="SP1"
-	align="horiz. centered"
-}
-rectangle {
-	object {
-		x=110
-		y=27
-		width=10
-		height=10
-	}
-	"basic attribute" {
-		clr=30
-	}
-	"dynamic attribute" {
-		vis="calc"
-		calc="A>B"
-		chan="$(P)$(Q):Value"
-		chanB="$(P)$(Q):SetPt1"
-	}
-}
-text {
-	object {
-		x=128
-		y=40
-		width=20
-		height=10
-	}
-	"basic attribute" {
-		clr=0
-		fill="outline"
-	}
-	textix="SP2"
-	align="horiz. centered"
-}
-rectangle {
-	object {
-		x=133
-		y=27
-		width=10
-		height=10
-	}
-	"basic attribute" {
-		clr=16
-	}
-	"dynamic attribute" {
-		vis="calc"
-		calc="A>B"
-		chan="$(P)$(Q):Value"
-		chanB="$(P)$(Q):SetPt2"
-	}
-}
-text {
-	object {
-		x=155
-		y=40
-		width=20
-		height=10
-	}
-	"basic attribute" {
-		clr=0
-		fill="outline"
-	}
-	textix="Alrm"
-	align="horiz. centered"
-}
-rectangle {
-	object {
-		x=160
-		y=27
-		width=10
-		height=10
-	}
-	"basic attribute" {
-		clr=20
-	}
-	"dynamic attribute" {
-		vis="calc"
-		calc="(A<B)||(A>C)"
-		chan="$(P)$(Q):Value"
-		chanB="$(P)$(Q):AlarmLo"
-		chanC="$(P)$(Q):AlarmHi"
-	}
-}
-"related display" {
-	object {
-		x=167
-		y=4
-		width=12
-		height=12
-	}
-	display[0] {
-		label="Love Control"
-		name="LoveControllerControl.adl"
-		args="P=$(P),Q=$(Q)"
-	}
-	clr=8
-	bclr=7
-	visual="invisible"
-}
diff --git a/optics/documentation/fb_epid/source/fb_epid.adl b/optics/documentation/fb_epid/source/fb_epid.adl
deleted file mode 100644
index 8351941..0000000
--- a/optics/documentation/fb_epid/source/fb_epid.adl
+++ /dev/null
@@ -1,1194 +0,0 @@
-
-file {
-	name="opticsApp/op/adl/fb_epid.adl"
-	version=030003
-}
-display {
-	object {
-		x=26
-		y=59
-		width=518
-		height=373
-	}
-	clr=14
-	bclr=57
-	cmap=""
-	gridSpacing=5
-	gridOn=0
-	snapToGrid=0
-}
-"color map" {
-	ncolors=65
-	colors {
-		ffffff,
-		ececec,
-		dadada,
-		c8c8c8,
-		bbbbbb,
-		aeaeae,
-		9e9e9e,
-		919191,
-		858585,
-		787878,
-		696969,
-		5a5a5a,
-		464646,
-		2d2d2d,
-		000000,
-		00d800,
-		1ebb00,
-		339900,
-		2d7f00,
-		216c00,
-		fd0000,
-		de1309,
-		be190b,
-		a01207,
-		820400,
-		5893ff,
-		597ee1,
-		4b6ec7,
-		3a5eab,
-		27548d,
-		fbf34a,
-		f9da3c,
-		eeb62b,
-		e19015,
-		cd6100,
-		ffb0ff,
-		d67fe2,
-		ae4ebc,
-		8b1a96,
-		610a75,
-		a4aaff,
-		8793e2,
-		6a73c1,
-		4d52a4,
-		343386,
-		c7bb6d,
-		b79d5c,
-		a47e3c,
-		7d5627,
-		58340f,
-		99ffff,
-		73dfff,
-		4ea5f9,
-		2a63e4,
-		0a00b8,
-		ebf1b5,
-		d4db9d,
-		bbc187,
-		a6a462,
-		8b8239,
-		73ff6b,
-		52da3b,
-		3cb420,
-		289315,
-		1a7309,
-	}
-}
-rectangle {
-	object {
-		x=8
-		y=90
-		width=500
-		height=92
-	}
-	"basic attribute" {
-		clr=14
-		fill="outline"
-		width=4
-	}
-}
-rectangle {
-	object {
-		x=37
-		y=85
-		width=140
-		height=14
-	}
-	"basic attribute" {
-		clr=57
-	}
-}
-text {
-	object {
-		x=16
-		y=4
-		width=200
-		height=10
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="fb_epid.adl"
-}
-text {
-	object {
-		x=16
-		y=14
-		width=200
-		height=10
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="last update: 2005-06-16 (prj)"
-}
-text {
-	object {
-		x=164
-		y=3
-		width=340
-		height=24
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="feedback   $(P)"
-	align="horiz. centered"
-}
-text {
-	object {
-		x=17
-		y=35
-		width=31
-		height=16
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="DESC"
-	align="horiz. right"
-}
-"text entry" {
-	object {
-		x=55
-		y=34
-		width=217
-		height=18
-	}
-	control {
-		chan="$(P).DESC"
-		clr=14
-		bclr=50
-	}
-	limits {
-	}
-}
-text {
-	object {
-		x=277
-		y=35
-		width=27
-		height=16
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="EGU"
-	align="horiz. right"
-}
-"text entry" {
-	object {
-		x=310
-		y=34
-		width=92
-		height=18
-	}
-	control {
-		chan="$(P).EGU"
-		clr=14
-		bclr=56
-	}
-	limits {
-	}
-}
-text {
-	object {
-		x=408
-		y=36
-		width=38
-		height=14
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="PREC"
-	align="horiz. right"
-}
-"text entry" {
-	object {
-		x=451
-		y=35
-		width=46
-		height=16
-	}
-	control {
-		chan="$(P).PREC"
-		clr=14
-		bclr=58
-	}
-	limits {
-	}
-}
-text {
-	object {
-		x=17
-		y=59
-		width=32
-		height=14
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="CT"
-	align="horiz. right"
-}
-"text update" {
-	object {
-		x=54
-		y=59
-		width=90
-		height=14
-	}
-	monitor {
-		chan="$(P).CT"
-		clr=14
-		bclr=58
-	}
-	align="horiz. centered"
-	limits {
-	}
-}
-text {
-	object {
-		x=149
-		y=59
-		width=32
-		height=14
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="DT"
-	align="horiz. right"
-}
-"text entry" {
-	object {
-		x=186
-		y=58
-		width=90
-		height=16
-	}
-	control {
-		chan="$(P).DT"
-		clr=14
-		bclr=58
-	}
-	limits {
-	}
-}
-menu {
-	object {
-		x=282
-		y=59
-		width=83
-		height=14
-	}
-	control {
-		chan="$(P).SCAN"
-		clr=14
-		bclr=1
-	}
-}
-"text entry" {
-	object {
-		x=407
-		y=58
-		width=90
-		height=16
-	}
-	control {
-		chan="$(P).MDT"
-		clr=14
-		bclr=58
-	}
-	limits {
-	}
-}
-text {
-	object {
-		x=370
-		y=59
-		width=32
-		height=14
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="MDT"
-	align="horiz. right"
-}
-rectangle {
-	object {
-		x=8
-		y=200
-		width=500
-		height=130
-	}
-	"basic attribute" {
-		clr=14
-		fill="outline"
-		width=4
-	}
-}
-text {
-	object {
-		x=47
-		y=85
-		width=120
-		height=14
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="EPID input"
-	align="horiz. centered"
-}
-rectangle {
-	object {
-		x=37
-		y=195
-		width=140
-		height=14
-	}
-	"basic attribute" {
-		clr=57
-	}
-}
-text {
-	object {
-		x=47
-		y=195
-		width=120
-		height=14
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="EPID output"
-	align="horiz. centered"
-}
-"text update" {
-	object {
-		x=49
-		y=127
-		width=102
-		height=14
-	}
-	monitor {
-		chan="$(P).CVAL"
-		clr=14
-		bclr=50
-	}
-	align="horiz. centered"
-	format="compact"
-	limits {
-	}
-}
-"choice button" {
-	object {
-		x=339
-		y=212
-		width=156
-		height=14
-	}
-	control {
-		chan="$(P).FMOD"
-		clr=14
-		bclr=56
-	}
-	stacking="column"
-}
-polyline {
-	object {
-		x=141
-		y=293
-		width=124
-		height=2
-	}
-	"basic attribute" {
-		clr=14
-		width=2
-	}
-	points {
-		(142,294)
-		(264,294)
-	}
-}
-"text update" {
-	object {
-		x=153
-		y=301
-		width=100
-		height=14
-	}
-	monitor {
-		chan="$(P).OVAL"
-		clr=14
-		bclr=50
-	}
-	align="horiz. centered"
-	format="compact"
-	limits {
-	}
-}
-text {
-	object {
-		x=124
-		y=301
-		width=24
-		height=14
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="<="
-	align="horiz. centered"
-}
-text {
-	object {
-		x=258
-		y=301
-		width=24
-		height=14
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="<="
-	align="horiz. centered"
-}
-"text entry" {
-	object {
-		x=39
-		y=300
-		width=80
-		height=16
-	}
-	control {
-		chan="$(P).DRVL"
-		clr=14
-		bclr=56
-	}
-	format="compact"
-	limits {
-	}
-}
-"text entry" {
-	object {
-		x=287
-		y=300
-		width=80
-		height=16
-	}
-	control {
-		chan="$(P).DRVH"
-		clr=14
-		bclr=56
-	}
-	format="compact"
-	limits {
-	}
-}
-text {
-	object {
-		x=249
-		y=234
-		width=12
-		height=14
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="P"
-	align="horiz. centered"
-}
-"text update" {
-	object {
-		x=164
-		y=234
-		width=80
-		height=14
-	}
-	monitor {
-		chan="$(E).P"
-		clr=14
-		bclr=50
-	}
-	align="horiz. centered"
-	format="compact"
-	limits {
-	}
-}
-text {
-	object {
-		x=145
-		y=234
-		width=14
-		height=14
-	}
-	"basic attribute" {
-		clr=14
-	}
-	align="horiz. centered"
-}
-text {
-	object {
-		x=249
-		y=274
-		width=12
-		height=14
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="D"
-	align="horiz. centered"
-}
-text {
-	object {
-		x=145
-		y=274
-		width=14
-		height=14
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="+"
-	align="horiz. centered"
-}
-text {
-	object {
-		x=249
-		y=253
-		width=12
-		height=14
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="I"
-	align="horiz. centered"
-}
-text {
-	object {
-		x=145
-		y=253
-		width=14
-		height=14
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="+"
-	align="horiz. centered"
-}
-"text entry" {
-	object {
-		x=38
-		y=232
-		width=80
-		height=16
-	}
-	control {
-		chan="$(E).KP"
-		clr=14
-		bclr=50
-	}
-	format="compact"
-	limits {
-	}
-}
-text {
-	object {
-		x=14
-		y=233
-		width=20
-		height=14
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="KP"
-	align="horiz. right"
-}
-"text entry" {
-	object {
-		x=38
-		y=253
-		width=80
-		height=16
-	}
-	control {
-		chan="$(P).KI"
-		clr=14
-		bclr=50
-	}
-	format="compact"
-	limits {
-	}
-}
-text {
-	object {
-		x=15
-		y=254
-		width=20
-		height=14
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="KI"
-	align="horiz. right"
-}
-"text entry" {
-	object {
-		x=39
-		y=274
-		width=80
-		height=16
-	}
-	control {
-		chan="$(P).KD"
-		clr=14
-		bclr=50
-	}
-	format="compact"
-	limits {
-	}
-}
-text {
-	object {
-		x=14
-		y=274
-		width=20
-		height=14
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="KD"
-	align="horiz. right"
-}
-"text entry" {
-	object {
-		x=38
-		y=232
-		width=80
-		height=16
-	}
-	control {
-		chan="$(P).KP"
-		clr=14
-		bclr=50
-	}
-	format="compact"
-	limits {
-	}
-}
-text {
-	object {
-		x=14
-		y=233
-		width=20
-		height=14
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="KP"
-	align="horiz. right"
-}
-text {
-	object {
-		x=249
-		y=253
-		width=12
-		height=14
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="I"
-	align="horiz. centered"
-}
-text {
-	object {
-		x=145
-		y=253
-		width=14
-		height=14
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="+"
-	align="horiz. centered"
-}
-text {
-	object {
-		x=249
-		y=234
-		width=12
-		height=14
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="P"
-	align="horiz. centered"
-}
-text {
-	object {
-		x=145
-		y=234
-		width=14
-		height=14
-	}
-	"basic attribute" {
-		clr=14
-	}
-	align="horiz. centered"
-}
-text {
-	object {
-		x=249
-		y=274
-		width=12
-		height=14
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="D"
-	align="horiz. centered"
-}
-text {
-	object {
-		x=145
-		y=274
-		width=14
-		height=14
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="+"
-	align="horiz. centered"
-}
-text {
-	object {
-		x=125
-		y=213
-		width=60
-		height=12
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="feedback"
-}
-"choice button" {
-	object {
-		x=40
-		y=212
-		width=80
-		height=14
-	}
-	control {
-		chan="$(P):on"
-		clr=14
-		bclr=50
-	}
-	stacking="column"
-}
-"text entry" {
-	object {
-		x=49
-		y=105
-		width=103
-		height=16
-	}
-	control {
-		chan="$(P).VAL"
-		clr=14
-		bclr=50
-	}
-	format="compact"
-	limits {
-	}
-}
-text {
-	object {
-		x=156
-		y=105
-		width=160
-		height=14
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="set point (VAL)"
-}
-text {
-	object {
-		x=17
-		y=127
-		width=14
-		height=14
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="-"
-	align="horiz. centered"
-}
-polyline {
-	object {
-		x=36
-		y=147
-		width=121
-		height=2
-	}
-	"basic attribute" {
-		clr=14
-		width=2
-	}
-	points {
-		(37,148)
-		(156,148)
-	}
-}
-text {
-	object {
-		x=156
-		y=154
-		width=160
-		height=14
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="following error (FE)"
-}
-"text update" {
-	object {
-		x=49
-		y=154
-		width=102
-		height=14
-	}
-	monitor {
-		chan="$(P).ERR"
-		clr=14
-		bclr=50
-	}
-	align="horiz. centered"
-	format="compact"
-	limits {
-	}
-}
-"related display" {
-	object {
-		x=156
-		y=126
-		width=160
-		height=18
-	}
-	display[0] {
-		label="calculation"
-		name="userCalc_full.adl"
-		args="P=$(P):,C=in"
-	}
-	clr=0
-	bclr=59
-	label="-input calc"
-}
-"related display" {
-	object {
-		x=370
-		y=299
-		width=120
-		height=18
-	}
-	display[0] {
-		label="calculation"
-		name="userCalc_full.adl"
-		args="P=$(P):,C=out"
-	}
-	clr=0
-	bclr=59
-	label="-output calc"
-}
-"related display" {
-	object {
-		x=132
-		y=341
-		width=120
-		height=18
-	}
-	display[0] {
-		label="calculation"
-		name="userCalc_full.adl"
-		args="P=$(P):,C=obuf"
-	}
-	clr=0
-	bclr=59
-	label="-output buffer calc"
-}
-"related display" {
-	object {
-		x=257
-		y=341
-		width=120
-		height=18
-	}
-	display[0] {
-		label="calculation"
-		name="userCalc_full.adl"
-		args="P=$(P):,C=out"
-	}
-	clr=0
-	bclr=59
-	label="-output calc"
-}
-"related display" {
-	object {
-		x=382
-		y=341
-		width=120
-		height=18
-	}
-	display[0] {
-		label="calculation"
-		name="userCalc_full.adl"
-		args="P=$(P):,C=resume"
-	}
-	clr=0
-	bclr=59
-	label="-resume calc"
-}
-"related display" {
-	object {
-		x=7
-		y=341
-		width=120
-		height=18
-	}
-	display[0] {
-		label="calculation"
-		name="userCalc_full.adl"
-		args="P=$(P):,C=enable"
-	}
-	clr=0
-	bclr=59
-	label="-'enable' calc"
-}
-"text update" {
-	object {
-		x=236
-		y=211
-		width=80
-		height=14
-	}
-	monitor {
-		chan="$(P).FBON"
-		clr=14
-		bclr=50
-	}
-	align="horiz. centered"
-	format="compact"
-	limits {
-	}
-}
-"text update" {
-	object {
-		x=164
-		y=234
-		width=80
-		height=14
-	}
-	monitor {
-		chan="$(P).P"
-		clr=14
-		bclr=50
-	}
-	align="horiz. centered"
-	format="compact"
-	limits {
-	}
-}
-composite {
-	object {
-		x=164
-		y=236
-		width=338
-		height=52
-	}
-	"composite name"=""
-	children {
-		"text update" {
-			object {
-				x=164
-				y=274
-				width=80
-				height=14
-			}
-			monitor {
-				chan="$(P).D"
-				clr=14
-				bclr=50
-			}
-			align="horiz. centered"
-			format="compact"
-			limits {
-			}
-		}
-		"text entry" {
-			object {
-				x=164
-				y=253
-				width=80
-				height=16
-			}
-			control {
-				chan="$(P).I"
-				clr=14
-				bclr=50
-			}
-			format="compact"
-			limits {
-			}
-		}
-		text {
-			object {
-				x=282
-				y=256
-				width=220
-				height=10
-			}
-			"basic attribute" {
-				clr=14
-			}
-			textix="I = KP * KI * sum(FE*DT)"
-		}
-		text {
-			object {
-				x=282
-				y=236
-				width=220
-				height=10
-			}
-			"basic attribute" {
-				clr=14
-			}
-			textix="P = KP * FE"
-		}
-		text {
-			object {
-				x=282
-				y=277
-				width=220
-				height=10
-			}
-			"basic attribute" {
-				clr=14
-			}
-			textix="D = KP * KD * (FE[i] - FE[i-1])/DT"
-		}
-	}
-	"dynamic attribute" {
-		vis="if zero"
-		chan="$(P).FMOD"
-	}
-}
-composite {
-	object {
-		x=199
-		y=236
-		width=303
-		height=52
-	}
-	"composite name"=""
-	children {
-		text {
-			object {
-				x=199
-				y=253
-				width=12
-				height=14
-			}
-			"basic attribute" {
-				clr=14
-			}
-			textix="0"
-			align="horiz. centered"
-		}
-		text {
-			object {
-				x=199
-				y=274
-				width=12
-				height=14
-			}
-			"basic attribute" {
-				clr=14
-			}
-			textix="0"
-			align="horiz. centered"
-		}
-		text {
-			object {
-				x=282
-				y=277
-				width=220
-				height=10
-			}
-			"basic attribute" {
-				clr=14
-			}
-			textix="D = 0"
-		}
-		text {
-			object {
-				x=282
-				y=256
-				width=220
-				height=10
-			}
-			"basic attribute" {
-				clr=14
-			}
-			textix="I = 0"
-		}
-		text {
-			object {
-				x=282
-				y=236
-				width=220
-				height=10
-			}
-			"basic attribute" {
-				clr=14
-			}
-			textix="P = KP * (INP[i] - INP[i-1])/DT"
-		}
-	}
-	"dynamic attribute" {
-		vis="if not zero"
-		chan="$(P).FMOD"
-	}
-}
diff --git a/sscan/sscanApp/op/opi/arrayPlot.opi b/sscan/sscanApp/op/opi/arrayPlot.opi
deleted file mode 100644
index c39cfed..0000000
--- a/sscan/sscanApp/op/opi/arrayPlot.opi
+++ /dev/null
@@ -1,684 +0,0 @@
-<display typeId="org.csstudio.opibuilder.Display" version="1.0">
-  <auto_zoom_to_fit_all>false</auto_zoom_to_fit_all>
-  <macros>
-    <include_parent_macros>true</include_parent_macros>
-  </macros>
-  <wuid>39872d87:13e14015f61:61f1</wuid>
-  <boy_version>3.1.2.20120830</boy_version>
-  <scripts />
-  <show_ruler>true</show_ruler>
-  <height>235</height>
-  <name>arrayPlot</name>
-  <snap_to_geometry>false</snap_to_geometry>
-  <show_grid>false</show_grid>
-  <background_color>
-    <color name="Gray_0" red="255" green="255" blue="255" />
-  </background_color>
-  <foreground_color>
-    <color name="Gray_14" red="0" green="0" blue="0" />
-  </foreground_color>
-  <widget_type>Display</widget_type>
-  <show_close_button>true</show_close_button>
-  <width>400</width>
-  <rules />
-  <show_edit_range>true</show_edit_range>
-  <grid_space>5</grid_space>
-  <auto_scale_widgets>
-    <auto_scale_widgets>false</auto_scale_widgets>
-    <min_width>-1</min_width>
-    <min_height>-1</min_height>
-  </auto_scale_widgets>
-  <actions hook="false" hook_all="false" />
-  <y>571</y>
-  <x>578</x>
-  <widget typeId="org.csstudio.opibuilder.widgets.xyGraph" version="1.0">
-    <trace_1_visible>true</trace_1_visible>
-    <trace_14_concatenate_data>true</trace_14_concatenate_data>
-    <trace_9_y_pv></trace_9_y_pv>
-    <trace_6_update_delay>100</trace_6_update_delay>
-    <trace_14_update_mode>0</trace_14_update_mode>
-    <trace_1_trace_type>0</trace_1_trace_type>
-    <trace_16_buffer_size>100</trace_16_buffer_size>
-    <trace_12_y_axis_index>1</trace_12_y_axis_index>
-    <trace_4_point_style>0</trace_4_point_style>
-    <axis_3_dash_grid_line>true</axis_3_dash_grid_line>
-    <trace_17_x_pv_value />
-    <trace_0_trace_color>
-      <color red="0" green="0" blue="0" />
-    </trace_0_trace_color>
-    <trace_18_point_style>0</trace_18_point_style>
-    <trace_19_buffer_size>100</trace_19_buffer_size>
-    <trace_10_point_style>0</trace_10_point_style>
-    <axis_0_auto_scale>true</axis_0_auto_scale>
-    <trace_3_trace_type>0</trace_3_trace_type>
-    <axis_3_minimum>0.0</axis_3_minimum>
-    <trace_18_y_pv></trace_18_y_pv>
-    <trace_18_anti_alias>true</trace_18_anti_alias>
-    <trace_14_y_pv_value />
-    <trace_9_buffer_size>100</trace_9_buffer_size>
-    <trace_4_update_delay>100</trace_4_update_delay>
-    <axis_1_time_format>0</axis_1_time_format>
-    <trace_19_update_delay>100</trace_19_update_delay>
-    <trace_11_point_style>0</trace_11_point_style>
-    <trace_1_point_size>4</trace_1_point_size>
-    <trace_10_y_axis_index>1</trace_10_y_axis_index>
-    <trace_0_plot_mode>0</trace_0_plot_mode>
-    <trace_15_trace_type>0</trace_15_trace_type>
-    <trace_13_update_mode>0</trace_13_update_mode>
-    <trace_18_buffer_size>100</trace_18_buffer_size>
-    <trace_count>1</trace_count>
-    <trace_4_concatenate_data>true</trace_4_concatenate_data>
-    <trace_2_y_pv_value />
-    <trace_14_visible>true</trace_14_visible>
-    <trace_13_x_pv></trace_13_x_pv>
-    <axis_0_show_grid>true</axis_0_show_grid>
-    <axis_2_log_scale>false</axis_2_log_scale>
-    <trace_18_concatenate_data>true</trace_18_concatenate_data>
-    <trace_16_x_pv_value />
-    <trace_11_line_width>1</trace_11_line_width>
-    <axis_0_log_scale>false</axis_0_log_scale>
-    <trace_11_x_axis_index>0</trace_11_x_axis_index>
-    <trace_3_point_size>4</trace_3_point_size>
-    <trace_19_name>$(trace_19_y_pv)</trace_19_name>
-    <trace_9_visible>true</trace_9_visible>
-    <trace_10_concatenate_data>true</trace_10_concatenate_data>
-    <trace_11_update_mode>0</trace_11_update_mode>
-    <trace_10_line_width>1</trace_10_line_width>
-    <trace_18_x_axis_index>0</trace_18_x_axis_index>
-    <trace_6_trace_type>0</trace_6_trace_type>
-    <trace_3_y_axis_index>1</trace_3_y_axis_index>
-    <trace_7_point_style>0</trace_7_point_style>
-    <trace_1_name>$(trace_1_y_pv)</trace_1_name>
-    <trace_17_name>$(trace_17_y_pv)</trace_17_name>
-    <trace_12_concatenate_data>true</trace_12_concatenate_data>
-    <axis_2_grid_color>
-      <color red="200" green="200" blue="200" />
-    </axis_2_grid_color>
-    <trace_12_point_style>0</trace_12_point_style>
-    <axis_2_minimum>0.0</axis_2_minimum>
-    <trace_4_line_width>1</trace_4_line_width>
-    <trace_0_point_size>4</trace_0_point_size>
-    <trace_14_x_pv_value />
-    <axis_3_show_grid>true</axis_3_show_grid>
-    <trace_14_point_style>0</trace_14_point_style>
-    <trace_13_point_style>0</trace_13_point_style>
-    <show_legend>true</show_legend>
-    <trace_15_x_axis_index>0</trace_15_x_axis_index>
-    <trace_8_line_width>1</trace_8_line_width>
-    <trace_10_trace_color>
-      <color red="114" green="40" blue="3" />
-    </trace_10_trace_color>
-    <y>15</y>
-    <trace_5_update_mode>0</trace_5_update_mode>
-    <x>0</x>
-    <axis_3_axis_color>
-      <color red="0" green="0" blue="0" />
-    </axis_3_axis_color>
-    <visible>true</visible>
-    <trace_0_x_pv_value />
-    <trace_11_x_pv></trace_11_x_pv>
-    <axis_1_visible>true</axis_1_visible>
-    <trace_13_y_pv_value />
-    <trace_9_x_pv_value />
-    <height>200</height>
-    <axis_1_maximum>100.0</axis_1_maximum>
-    <trace_10_name>$(trace_10_y_pv)</trace_10_name>
-    <trace_3_x_pv_value />
-    <trace_17_trace_color>
-      <color red="255" green="170" blue="0" />
-    </trace_17_trace_color>
-    <trace_9_name>$(trace_9_y_pv)</trace_9_name>
-    <trace_8_buffer_size>100</trace_8_buffer_size>
-    <trace_16_plot_mode>0</trace_16_plot_mode>
-    <trace_9_plot_mode>0</trace_9_plot_mode>
-    <trace_17_anti_alias>true</trace_17_anti_alias>
-    <trace_12_line_width>1</trace_12_line_width>
-    <trace_4_x_pv></trace_4_x_pv>
-    <trace_9_trace_color>
-      <color red="0" green="214" blue="255" />
-    </trace_9_trace_color>
-    <trace_2_x_pv></trace_2_x_pv>
-    <trace_7_update_delay>100</trace_7_update_delay>
-    <axis_1_title_font>
-      <fontdata fontName="Arial" height="9" style="1" />
-    </axis_1_title_font>
-    <border_width>1</border_width>
-    <axis_2_left_bottom_side>true</axis_2_left_bottom_side>
-    <trace_2_x_axis_index>0</trace_2_x_axis_index>
-    <axis_2_show_grid>true</axis_2_show_grid>
-    <trace_3_name>$(trace_3_y_pv)</trace_3_name>
-    <trace_19_plot_mode>0</trace_19_plot_mode>
-    <trace_19_y_pv_value />
-    <axis_0_axis_color>
-      <color red="0" green="0" blue="0" />
-    </axis_0_axis_color>
-    <trace_13_trace_type>0</trace_13_trace_type>
-    <trace_14_trace_type>0</trace_14_trace_type>
-    <axis_1_scale_format></axis_1_scale_format>
-    <trace_13_buffer_size>100</trace_13_buffer_size>
-    <name>XY Graph</name>
-    <trace_11_trace_type>0</trace_11_trace_type>
-    <axis_1_auto_scale_threshold>0.0</axis_1_auto_scale_threshold>
-    <trace_19_concatenate_data>true</trace_19_concatenate_data>
-    <trace_4_buffer_size>100</trace_4_buffer_size>
-    <trace_10_y_pv_value />
-    <trace_0_line_width>1</trace_0_line_width>
-    <trace_16_anti_alias>true</trace_16_anti_alias>
-    <trace_19_line_width>1</trace_19_line_width>
-    <trace_19_x_axis_index>0</trace_19_x_axis_index>
-    <trace_12_x_axis_index>0</trace_12_x_axis_index>
-    <trace_11_plot_mode>0</trace_11_plot_mode>
-    <trace_2_x_pv_value />
-    <axis_3_auto_scale>true</axis_3_auto_scale>
-    <show_toolbar>true</show_toolbar>
-    <trace_13_x_pv_value />
-    <trace_15_buffer_size>100</trace_15_buffer_size>
-    <trace_1_update_delay>100</trace_1_update_delay>
-    <trace_4_x_axis_index>0</trace_4_x_axis_index>
-    <axis_1_show_grid>true</axis_1_show_grid>
-    <trace_9_x_axis_index>0</trace_9_x_axis_index>
-    <trace_15_name>$(trace_15_y_pv)</trace_15_name>
-    <axis_0_grid_color>
-      <color red="200" green="200" blue="200" />
-    </axis_0_grid_color>
-    <trace_5_plot_mode>0</trace_5_plot_mode>
-    <trace_17_update_delay>100</trace_17_update_delay>
-    <trace_19_anti_alias>true</trace_19_anti_alias>
-    <trace_2_line_width>1</trace_2_line_width>
-    <trace_19_x_pv_value />
-    <trace_1_line_width>1</trace_1_line_width>
-    <trace_18_plot_mode>0</trace_18_plot_mode>
-    <trace_2_y_axis_index>1</trace_2_y_axis_index>
-    <axis_0_auto_scale_threshold>0.0</axis_0_auto_scale_threshold>
-    <trace_15_point_style>0</trace_15_point_style>
-    <trace_2_trace_type>0</trace_2_trace_type>
-    <trace_0_name>$(trace_0_y_pv)</trace_0_name>
-    <trace_1_anti_alias>true</trace_1_anti_alias>
-    <axis_3_auto_scale_threshold>0.0</axis_3_auto_scale_threshold>
-    <trace_3_x_pv></trace_3_x_pv>
-    <trace_11_y_axis_index>1</trace_11_y_axis_index>
-    <trace_13_point_size>4</trace_13_point_size>
-    <axis_2_maximum>100.0</axis_2_maximum>
-    <trace_0_update_mode>0</trace_0_update_mode>
-    <trace_19_visible>true</trace_19_visible>
-    <trace_15_concatenate_data>true</trace_15_concatenate_data>
-    <trace_17_y_axis_index>1</trace_17_y_axis_index>
-    <trace_10_y_pv></trace_10_y_pv>
-    <trace_9_x_pv></trace_9_x_pv>
-    <trace_1_buffer_size>100</trace_1_buffer_size>
-    <axis_0_axis_title></axis_0_axis_title>
-    <trace_3_line_width>1</trace_3_line_width>
-    <trace_11_trace_color>
-      <color red="219" green="128" blue="4" />
-    </trace_11_trace_color>
-    <trace_14_name>$(trace_14_y_pv)</trace_14_name>
-    <wuid>39872d87:13e14015f61:61f2</wuid>
-    <trace_2_buffer_size>100</trace_2_buffer_size>
-    <trace_18_trace_color>
-      <color red="255" green="0" blue="240" />
-    </trace_18_trace_color>
-    <axis_1_axis_color>
-      <color red="0" green="0" blue="0" />
-    </axis_1_axis_color>
-    <trace_19_point_size>4</trace_19_point_size>
-    <trace_2_update_mode>0</trace_2_update_mode>
-    <foreground_color>
-      <color red="0" green="0" blue="255" />
-    </foreground_color>
-    <trace_15_plot_mode>0</trace_15_plot_mode>
-    <trace_1_update_mode>0</trace_1_update_mode>
-    <backcolor_alarm_sensitive>false</backcolor_alarm_sensitive>
-    <axis_1_axis_title>$(YLABEL)</axis_1_axis_title>
-    <trace_4_anti_alias>true</trace_4_anti_alias>
-    <trace_12_x_pv_value />
-    <trace_9_line_width>1</trace_9_line_width>
-    <trace_7_x_pv_value />
-    <trace_4_plot_mode>0</trace_4_plot_mode>
-    <axis_2_y_axis>true</axis_2_y_axis>
-    <trace_3_update_mode>0</trace_3_update_mode>
-    <axis_0_maximum>100.0</axis_0_maximum>
-    <trace_14_update_delay>100</trace_14_update_delay>
-    <trace_7_anti_alias>true</trace_7_anti_alias>
-    <trace_9_point_size>4</trace_9_point_size>
-    <trace_12_update_delay>100</trace_12_update_delay>
-    <trace_19_trace_type>0</trace_19_trace_type>
-    <trace_16_name>$(trace_16_y_pv)</trace_16_name>
-    <axis_0_scale_font>
-      <opifont.name fontName="Sans" height="10" style="0">Default</opifont.name>
-    </axis_0_scale_font>
-    <trace_19_y_axis_index>1</trace_19_y_axis_index>
-    <trace_17_trace_type>0</trace_17_trace_type>
-    <trace_12_visible>true</trace_12_visible>
-    <trace_12_name>$(trace_12_y_pv)</trace_12_name>
-    <trace_2_point_size>4</trace_2_point_size>
-    <trace_12_plot_mode>0</trace_12_plot_mode>
-    <trace_16_trace_type>0</trace_16_trace_type>
-    <trace_4_y_axis_index>1</trace_4_y_axis_index>
-    <trace_12_update_mode>0</trace_12_update_mode>
-    <trace_6_plot_mode>0</trace_6_plot_mode>
-    <trace_10_x_axis_index>0</trace_10_x_axis_index>
-    <trace_1_y_pv_value />
-    <axis_0_time_format>0</axis_0_time_format>
-    <trace_8_name>$(trace_8_y_pv)</trace_8_name>
-    <trace_4_update_mode>0</trace_4_update_mode>
-    <trace_14_x_pv></trace_14_x_pv>
-    <trace_15_y_pv_value />
-    <trace_8_point_style>0</trace_8_point_style>
-    <trace_10_x_pv></trace_10_x_pv>
-    <trace_10_visible>true</trace_10_visible>
-    <trace_3_concatenate_data>true</trace_3_concatenate_data>
-    <trace_3_trace_color>
-      <color red="0" green="0" blue="0" />
-    </trace_3_trace_color>
-    <trace_15_y_pv></trace_15_y_pv>
-    <trace_11_y_pv_value />
-    <trace_1_plot_mode>0</trace_1_plot_mode>
-    <trace_8_visible>true</trace_8_visible>
-    <trace_8_x_pv_value />
-    <trace_7_visible>true</trace_7_visible>
-    <trace_5_name>$(trace_5_y_pv)</trace_5_name>
-    <trace_17_concatenate_data>true</trace_17_concatenate_data>
-    <trace_10_buffer_size>100</trace_10_buffer_size>
-    <trace_7_y_axis_index>1</trace_7_y_axis_index>
-    <trace_6_point_style>0</trace_6_point_style>
-    <trace_13_x_axis_index>0</trace_13_x_axis_index>
-    <axis_1_log_scale>false</axis_1_log_scale>
-    <trace_16_x_pv></trace_16_x_pv>
-    <trace_17_line_width>1</trace_17_line_width>
-    <trace_8_trace_type>0</trace_8_trace_type>
-    <trace_15_x_pv_value />
-    <trace_18_y_pv_value />
-    <trace_1_point_style>0</trace_1_point_style>
-    <trace_5_x_pv></trace_5_x_pv>
-    <trace_18_y_axis_index>1</trace_18_y_axis_index>
-    <axis_2_auto_scale_threshold>0.0</axis_2_auto_scale_threshold>
-    <trace_18_x_pv_value />
-    <trace_13_update_delay>100</trace_13_update_delay>
-    <trace_7_y_pv></trace_7_y_pv>
-    <trace_13_name>$(trace_13_y_pv)</trace_13_name>
-    <trace_11_x_pv_value />
-    <trace_11_anti_alias>true</trace_11_anti_alias>
-    <trace_5_point_style>0</trace_5_point_style>
-    <axis_3_maximum>100.0</axis_3_maximum>
-    <trace_2_plot_mode>0</trace_2_plot_mode>
-    <title_font>
-      <fontdata fontName="Arial" height="12" style="1" />
-    </title_font>
-    <trace_6_y_pv></trace_6_y_pv>
-    <trace_8_y_pv_value />
-    <trace_8_trace_color>
-      <color red="0" green="255" blue="11" />
-    </trace_8_trace_color>
-    <trace_3_buffer_size>100</trace_3_buffer_size>
-    <axis_1_dash_grid_line>true</axis_1_dash_grid_line>
-    <trace_12_buffer_size>100</trace_12_buffer_size>
-    <trace_10_update_mode>0</trace_10_update_mode>
-    <trace_13_anti_alias>true</trace_13_anti_alias>
-    <trace_16_visible>true</trace_16_visible>
-    <axis_2_dash_grid_line>true</axis_2_dash_grid_line>
-    <trace_15_anti_alias>true</trace_15_anti_alias>
-    <trace_15_update_mode>0</trace_15_update_mode>
-    <trace_8_update_mode>0</trace_8_update_mode>
-    <axis_0_visible>true</axis_0_visible>
-    <trace_14_plot_mode>0</trace_14_plot_mode>
-    <trace_10_plot_mode>0</trace_10_plot_mode>
-    <trace_14_y_pv></trace_14_y_pv>
-    <trace_19_y_pv></trace_19_y_pv>
-    <trace_6_point_size>4</trace_6_point_size>
-    <trace_14_trace_color>
-      <color red="33" green="179" blue="33" />
-    </trace_14_trace_color>
-    <forecolor_alarm_sensitive>false</forecolor_alarm_sensitive>
-    <trace_18_trace_type>0</trace_18_trace_type>
-    <trace_7_plot_mode>0</trace_7_plot_mode>
-    <trace_12_x_pv></trace_12_x_pv>
-    <trace_1_x_pv_value />
-    <trace_14_y_axis_index>1</trace_14_y_axis_index>
-    <trace_6_concatenate_data>true</trace_6_concatenate_data>
-    <axis_0_scale_format></axis_0_scale_format>
-    <trace_4_name>$(trace_4_y_pv)</trace_4_name>
-    <trace_15_line_width>1</trace_15_line_width>
-    <trace_11_concatenate_data>true</trace_11_concatenate_data>
-    <trace_6_visible>true</trace_6_visible>
-    <trace_7_buffer_size>100</trace_7_buffer_size>
-    <trace_12_trace_type>0</trace_12_trace_type>
-    <trace_4_trace_type>0</trace_4_trace_type>
-    <trace_0_x_pv>$(X)</trace_0_x_pv>
-    <trace_16_update_delay>100</trace_16_update_delay>
-    <trace_16_update_mode>0</trace_16_update_mode>
-    <trace_13_visible>true</trace_13_visible>
-    <trace_7_line_width>1</trace_7_line_width>
-    <trace_0_y_pv>$(Y)</trace_0_y_pv>
-    <trace_18_name>$(trace_18_y_pv)</trace_18_name>
-    <trace_12_y_pv></trace_12_y_pv>
-    <axis_3_visible>true</axis_3_visible>
-    <trace_19_x_pv></trace_19_x_pv>
-    <trace_5_point_size>4</trace_5_point_size>
-    <trace_18_x_pv></trace_18_x_pv>
-    <trace_18_point_size>4</trace_18_point_size>
-    <trace_6_x_pv_value />
-    <trace_6_update_mode>0</trace_6_update_mode>
-    <axis_0_title_font>
-      <fontdata fontName="Arial" height="9" style="1" />
-    </axis_0_title_font>
-    <trace_5_buffer_size>100</trace_5_buffer_size>
-    <trace_16_y_pv_value />
-    <trace_3_point_style>0</trace_3_point_style>
-    <axis_1_auto_scale>true</axis_1_auto_scale>
-    <trace_7_x_axis_index>0</trace_7_x_axis_index>
-    <trace_2_update_delay>100</trace_2_update_delay>
-    <trace_16_line_width>1</trace_16_line_width>
-    <trace_4_point_size>4</trace_4_point_size>
-    <trace_0_concatenate_data>true</trace_0_concatenate_data>
-    <trace_9_update_delay>100</trace_9_update_delay>
-    <trace_5_y_axis_index>1</trace_5_y_axis_index>
-    <trace_7_update_mode>0</trace_7_update_mode>
-    <trace_9_trace_type>0</trace_9_trace_type>
-    <trace_3_y_pv></trace_3_y_pv>
-    <actions hook="false" hook_all="false" />
-    <trigger_pv_value />
-    <trace_13_concatenate_data>true</trace_13_concatenate_data>
-    <trace_1_x_pv></trace_1_x_pv>
-    <trace_14_anti_alias>true</trace_14_anti_alias>
-    <show_plot_area_border>false</show_plot_area_border>
-    <trace_4_trace_color>
-      <color red="128" green="0" blue="255" />
-    </trace_4_trace_color>
-    <axis_2_axis_title>Secondary Axis (2)</axis_2_axis_title>
-    <trace_11_y_pv></trace_11_y_pv>
-    <trace_15_visible>true</trace_15_visible>
-    <trace_8_x_axis_index>0</trace_8_x_axis_index>
-    <widget_type>XY Graph</widget_type>
-    <enabled>true</enabled>
-    <width>400</width>
-    <trace_17_buffer_size>100</trace_17_buffer_size>
-    <trace_14_x_axis_index>0</trace_14_x_axis_index>
-    <axis_3_title_font>
-      <fontdata fontName="Arial" height="9" style="1" />
-    </axis_3_title_font>
-    <trace_13_y_axis_index>1</trace_13_y_axis_index>
-    <trace_16_x_axis_index>0</trace_16_x_axis_index>
-    <trace_0_buffer_size>100</trace_0_buffer_size>
-    <trace_11_update_delay>100</trace_11_update_delay>
-    <trigger_pv></trigger_pv>
-    <trace_19_point_style>0</trace_19_point_style>
-    <trace_6_trace_color>
-      <color red="255" green="0" blue="240" />
-    </trace_6_trace_color>
-    <trace_5_trace_type>0</trace_5_trace_type>
-    <trace_15_y_axis_index>1</trace_15_y_axis_index>
-    <trace_14_buffer_size>100</trace_14_buffer_size>
-    <axis_3_grid_color>
-      <color red="200" green="200" blue="200" />
-    </axis_3_grid_color>
-    <axis_count>2</axis_count>
-    <trace_2_name>$(trace_2_y_pv)</trace_2_name>
-    <trace_4_visible>true</trace_4_visible>
-    <trace_10_point_size>4</trace_10_point_size>
-    <trace_5_concatenate_data>true</trace_5_concatenate_data>
-    <transparent>false</transparent>
-    <trace_2_anti_alias>true</trace_2_anti_alias>
-    <trace_7_trace_type>0</trace_7_trace_type>
-    <axis_0_dash_grid_line>true</axis_0_dash_grid_line>
-    <trace_2_visible>true</trace_2_visible>
-    <trace_13_y_pv></trace_13_y_pv>
-    <trace_5_update_delay>100</trace_5_update_delay>
-    <trace_8_y_pv></trace_8_y_pv>
-    <trace_0_x_axis_index>0</trace_0_x_axis_index>
-    <trace_17_point_style>0</trace_17_point_style>
-    <trace_12_anti_alias>true</trace_12_anti_alias>
-    <trace_18_update_delay>100</trace_18_update_delay>
-    <trace_16_trace_color>
-      <color red="128" green="0" blue="255" />
-    </trace_16_trace_color>
-    <axis_2_time_format>0</axis_2_time_format>
-    <trace_6_y_axis_index>1</trace_6_y_axis_index>
-    <trace_17_point_size>4</trace_17_point_size>
-    <trace_6_y_pv_value />
-    <trace_0_point_style>0</trace_0_point_style>
-    <tooltip>$(trace_0_y_pv)
-$(trace_0_y_pv_value)</tooltip>
-    <trace_11_name>$(trace_11_y_pv)</trace_11_name>
-    <border_alarm_sensitive>true</border_alarm_sensitive>
-    <axis_2_title_font>
-      <fontdata fontName="Arial" height="9" style="1" />
-    </axis_2_title_font>
-    <trace_4_y_pv></trace_4_y_pv>
-    <trace_12_point_size>4</trace_12_point_size>
-    <trace_1_x_axis_index>0</trace_1_x_axis_index>
-    <trace_11_buffer_size>100</trace_11_buffer_size>
-    <trace_6_name>$(trace_6_y_pv)</trace_6_name>
-    <trace_8_point_size>4</trace_8_point_size>
-    <trace_19_update_mode>0</trace_19_update_mode>
-    <trace_8_y_axis_index>1</trace_8_y_axis_index>
-    <trace_16_concatenate_data>true</trace_16_concatenate_data>
-    <trace_9_point_style>0</trace_9_point_style>
-    <trace_0_y_axis_index>1</trace_0_y_axis_index>
-    <trace_3_visible>true</trace_3_visible>
-    <axis_1_scale_font>
-      <opifont.name fontName="Sans" height="10" style="0">Default</opifont.name>
-    </axis_1_scale_font>
-    <trace_17_update_mode>0</trace_17_update_mode>
-    <border_style>0</border_style>
-    <trace_13_line_width>1</trace_13_line_width>
-    <trace_3_x_axis_index>0</trace_3_x_axis_index>
-    <plot_area_background_color>
-      <color red="255" green="255" blue="255" />
-    </plot_area_background_color>
-    <trace_2_concatenate_data>true</trace_2_concatenate_data>
-    <trace_10_trace_type>0</trace_10_trace_type>
-    <axis_3_scale_font>
-      <opifont.name fontName="Sans" height="10" style="0">Default</opifont.name>
-    </axis_3_scale_font>
-    <trace_16_point_size>4</trace_16_point_size>
-    <trace_15_x_pv></trace_15_x_pv>
-    <trace_2_point_style>0</trace_2_point_style>
-    <axis_3_y_axis>true</axis_3_y_axis>
-    <trace_9_update_mode>0</trace_9_update_mode>
-    <title></title>
-    <axis_3_axis_title>Secondary Axis (3)</axis_3_axis_title>
-    <axis_3_log_scale>false</axis_3_log_scale>
-    <trace_8_anti_alias>true</trace_8_anti_alias>
-    <trace_13_plot_mode>0</trace_13_plot_mode>
-    <trace_10_update_delay>100</trace_10_update_delay>
-    <trace_1_y_pv></trace_1_y_pv>
-    <trace_8_x_pv></trace_8_x_pv>
-    <axis_2_auto_scale>true</axis_2_auto_scale>
-    <trace_17_plot_mode>0</trace_17_plot_mode>
-    <pv_name></pv_name>
-    <axis_1_minimum>0.0</axis_1_minimum>
-    <axis_2_axis_color>
-      <color red="0" green="0" blue="0" />
-    </axis_2_axis_color>
-    <trace_3_plot_mode>0</trace_3_plot_mode>
-    <trace_14_point_size>4</trace_14_point_size>
-    <trace_12_y_pv_value />
-    <trace_0_visible>true</trace_0_visible>
-    <trace_6_line_width>1</trace_6_line_width>
-    <axis_2_scale_font>
-      <opifont.name fontName="Sans" height="10" style="0">Default</opifont.name>
-    </axis_2_scale_font>
-    <trace_11_visible>true</trace_11_visible>
-    <axis_2_scale_format></axis_2_scale_format>
-    <trace_7_x_pv></trace_7_x_pv>
-    <axis_0_minimum>0.0</axis_0_minimum>
-    <trace_5_trace_color>
-      <color red="255" green="170" blue="0" />
-    </trace_5_trace_color>
-    <border_color>
-      <color red="0" green="128" blue="255" />
-    </border_color>
-    <trace_7_trace_color>
-      <color red="243" green="132" blue="132" />
-    </trace_7_trace_color>
-    <trace_16_y_pv></trace_16_y_pv>
-    <trace_8_update_delay>100</trace_8_update_delay>
-    <trace_11_point_size>4</trace_11_point_size>
-    <axis_3_left_bottom_side>true</axis_3_left_bottom_side>
-    <trace_3_update_delay>100</trace_3_update_delay>
-    <trace_9_concatenate_data>true</trace_9_concatenate_data>
-    <trace_9_y_axis_index>1</trace_9_y_axis_index>
-    <trace_8_concatenate_data>true</trace_8_concatenate_data>
-    <scripts />
-    <trace_5_visible>true</trace_5_visible>
-    <trace_0_anti_alias>true</trace_0_anti_alias>
-    <trace_4_x_pv_value />
-    <axis_3_time_format>0</axis_3_time_format>
-    <trace_13_trace_color>
-      <color red="242" green="26" blue="26" />
-    </trace_13_trace_color>
-    <trace_6_x_axis_index>0</trace_6_x_axis_index>
-    <trace_16_y_axis_index>1</trace_16_y_axis_index>
-    <trace_12_trace_color>
-      <color red="21" green="21" blue="196" />
-    </trace_12_trace_color>
-    <background_color>
-      <color red="240" green="240" blue="240" />
-    </background_color>
-    <trace_10_anti_alias>true</trace_10_anti_alias>
-    <trace_6_buffer_size>100</trace_6_buffer_size>
-    <trace_1_trace_color>
-      <color red="242" green="26" blue="26" />
-    </trace_1_trace_color>
-    <trace_2_trace_color>
-      <color red="33" green="179" blue="33" />
-    </trace_2_trace_color>
-    <trace_9_y_pv_value />
-    <trace_7_point_size>4</trace_7_point_size>
-    <trace_19_trace_color>
-      <color red="243" green="132" blue="132" />
-    </trace_19_trace_color>
-    <trace_6_anti_alias>true</trace_6_anti_alias>
-    <trace_17_x_axis_index>0</trace_17_x_axis_index>
-    <trace_5_y_pv_value />
-    <pv_value />
-    <trace_17_y_pv></trace_17_y_pv>
-    <trace_10_x_pv_value />
-    <trace_5_x_pv_value />
-    <trace_17_y_pv_value />
-    <trace_5_line_width>1</trace_5_line_width>
-    <trace_4_y_pv_value />
-    <trace_8_plot_mode>0</trace_8_plot_mode>
-    <trace_18_visible>true</trace_18_visible>
-    <trace_5_y_pv></trace_5_y_pv>
-    <trace_16_point_style>0</trace_16_point_style>
-    <trace_14_line_width>1</trace_14_line_width>
-    <trace_7_y_pv_value />
-    <trace_5_x_axis_index>0</trace_5_x_axis_index>
-    <axis_2_visible>true</axis_2_visible>
-    <trace_1_y_axis_index>1</trace_1_y_axis_index>
-    <trace_3_anti_alias>true</trace_3_anti_alias>
-    <trace_5_anti_alias>true</trace_5_anti_alias>
-    <axis_3_scale_format></axis_3_scale_format>
-    <scale_options>
-      <width_scalable>true</width_scalable>
-      <height_scalable>true</height_scalable>
-      <keep_wh_ratio>false</keep_wh_ratio>
-    </scale_options>
-    <trace_17_x_pv></trace_17_x_pv>
-    <trace_1_concatenate_data>true</trace_1_concatenate_data>
-    <trace_6_x_pv></trace_6_x_pv>
-    <trace_15_update_delay>100</trace_15_update_delay>
-    <trace_18_line_width>1</trace_18_line_width>
-    <trace_18_update_mode>0</trace_18_update_mode>
-    <rules />
-    <trace_2_y_pv></trace_2_y_pv>
-    <trace_0_y_pv_value />
-    <trace_0_trace_type>0</trace_0_trace_type>
-    <axis_1_grid_color>
-      <color red="200" green="200" blue="200" />
-    </axis_1_grid_color>
-    <trace_3_y_pv_value />
-    <trace_7_concatenate_data>true</trace_7_concatenate_data>
-    <trace_0_update_delay>100</trace_0_update_delay>
-    <trace_15_trace_color>
-      <color red="0" green="0" blue="0" />
-    </trace_15_trace_color>
-    <trace_9_anti_alias>true</trace_9_anti_alias>
-    <trace_15_point_size>4</trace_15_point_size>
-    <trace_17_visible>true</trace_17_visible>
-    <trace_7_name>$(trace_7_y_pv)</trace_7_name>
-  </widget>
-  <widget typeId="org.csstudio.opibuilder.widgets.Label" version="1.0">
-    <visible>true</visible>
-    <vertical_alignment>1</vertical_alignment>
-    <wuid>39872d87:13e14015f61:61f3</wuid>
-    <auto_size>false</auto_size>
-    <scripts />
-    <height>13</height>
-    <name>Label</name>
-    <scale_options>
-      <width_scalable>true</width_scalable>
-      <height_scalable>true</height_scalable>
-      <keep_wh_ratio>false</keep_wh_ratio>
-    </scale_options>
-    <transparent>true</transparent>
-    <show_scrollbar>false</show_scrollbar>
-    <background_color>
-      <color red="255" green="255" blue="255" />
-    </background_color>
-    <foreground_color>
-      <color name="Gray_14" red="0" green="0" blue="0" />
-    </foreground_color>
-    <widget_type>Label</widget_type>
-    <enabled>true</enabled>
-    <text>Y=$(Y)</text>
-    <font>
-      <fontdata fontName="Sans" height="8" style="0" />
-    </font>
-    <width>400</width>
-    <border_style>0</border_style>
-    <rules />
-    <border_width>1</border_width>
-    <border_color>
-      <color red="0" green="128" blue="255" />
-    </border_color>
-    <horizontal_alignment>1</horizontal_alignment>
-    <actions hook="false" hook_all="false" />
-    <y>0</y>
-    <wrap_words>true</wrap_words>
-    <tooltip></tooltip>
-    <x>0</x>
-  </widget>
-  <widget typeId="org.csstudio.opibuilder.widgets.Label" version="1.0">
-    <visible>true</visible>
-    <vertical_alignment>1</vertical_alignment>
-    <wuid>39872d87:13e14015f61:61f4</wuid>
-    <auto_size>false</auto_size>
-    <scripts />
-    <height>13</height>
-    <name>Label</name>
-    <scale_options>
-      <width_scalable>true</width_scalable>
-      <height_scalable>true</height_scalable>
-      <keep_wh_ratio>false</keep_wh_ratio>
-    </scale_options>
-    <transparent>true</transparent>
-    <show_scrollbar>false</show_scrollbar>
-    <background_color>
-      <color red="255" green="255" blue="255" />
-    </background_color>
-    <foreground_color>
-      <color name="Gray_14" red="0" green="0" blue="0" />
-    </foreground_color>
-    <widget_type>Label</widget_type>
-    <enabled>true</enabled>
-    <text>X=$(X)</text>
-    <font>
-      <fontdata fontName="Sans" height="8" style="0" />
-    </font>
-    <width>400</width>
-    <border_style>0</border_style>
-    <rules />
-    <border_width>1</border_width>
-    <border_color>
-      <color red="0" green="128" blue="255" />
-    </border_color>
-    <horizontal_alignment>1</horizontal_alignment>
-    <actions hook="false" hook_all="false" />
-    <y>218</y>
-    <wrap_words>true</wrap_words>
-    <tooltip></tooltip>
-    <x>0</x>
-  </widget>
-</display>
\ No newline at end of file
diff --git a/std/stdApp/op/adl/arrayPlot.adl b/std/stdApp/op/adl/arrayPlot.adl
deleted file mode 100644
index 7f9b3ac..0000000
--- a/std/stdApp/op/adl/arrayPlot.adl
+++ /dev/null
@@ -1,145 +0,0 @@
-
-file {
-	name="/home/beams/BCDA/epics/synApps_R3.13.2_R4.3/share/stdApp/op/adl/arrayPlot.adl"
-	version=020306
-}
-display {
-	object {
-		x=578
-		y=571
-		width=400
-		height=235
-	}
-	clr=14
-	bclr=0
-	cmap=""
-	gridSpacing=5
-	gridOn=0
-	snapToGrid=0
-}
-"color map" {
-	ncolors=65
-	colors {
-		ffffff,
-		ececec,
-		dadada,
-		c8c8c8,
-		bbbbbb,
-		aeaeae,
-		9e9e9e,
-		919191,
-		858585,
-		787878,
-		696969,
-		5a5a5a,
-		464646,
-		2d2d2d,
-		000000,
-		00d800,
-		1ebb00,
-		339900,
-		2d7f00,
-		216c00,
-		fd0000,
-		de1309,
-		be190b,
-		a01207,
-		820400,
-		5893ff,
-		597ee1,
-		4b6ec7,
-		3a5eab,
-		27548d,
-		fbf34a,
-		f9da3c,
-		eeb62b,
-		e19015,
-		cd6100,
-		ffb0ff,
-		d67fe2,
-		ae4ebc,
-		8b1a96,
-		610a75,
-		a4aaff,
-		8793e2,
-		6a73c1,
-		4d52a4,
-		343386,
-		c7bb6d,
-		b79d5c,
-		a47e3c,
-		7d5627,
-		58340f,
-		99ffff,
-		73dfff,
-		4ea5f9,
-		2a63e4,
-		0a00b8,
-		ebf1b5,
-		d4db9d,
-		bbc187,
-		a6a462,
-		8b8239,
-		73ff6b,
-		52da3b,
-		3cb420,
-		289315,
-		1a7309,
-	}
-}
-"cartesian plot" {
-	object {
-		x=0
-		y=15
-		width=400
-		height=200
-	}
-	plotcom {
-		ylabel="$(YLABEL)"
-		clr=14
-		bclr=0
-	}
-	style="line"
-	count="2000"
-	trace[0] {
-		xdata="$(X)"
-		ydata="$(Y)"
-		data_clr=14
-	}
-	x_axis {
-		rangeStyle="auto-scale"
-	}
-	y1_axis {
-		rangeStyle="auto-scale"
-	}
-	y2_axis {
-		rangeStyle="auto-scale"
-	}
-	eraseMode="if zero"
-}
-text {
-	object {
-		x=0
-		y=0
-		width=400
-		height=13
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="Y=$(Y)"
-	align="horiz. centered"
-}
-text {
-	object {
-		x=0
-		y=218
-		width=400
-		height=13
-	}
-	"basic attribute" {
-		clr=14
-	}
-	textix="X=$(X)"
-	align="horiz. centered"
-}
diff --git a/std/stdApp/op/opi/arrayPlot.opi b/std/stdApp/op/opi/arrayPlot.opi
deleted file mode 100644
index ffb9c03..0000000
--- a/std/stdApp/op/opi/arrayPlot.opi
+++ /dev/null
@@ -1,684 +0,0 @@
-<display typeId="org.csstudio.opibuilder.Display" version="1.0">
-  <auto_zoom_to_fit_all>false</auto_zoom_to_fit_all>
-  <macros>
-    <include_parent_macros>true</include_parent_macros>
-  </macros>
-  <wuid>39872d87:13e14015f61:7ec4</wuid>
-  <boy_version>3.1.2.20120830</boy_version>
-  <scripts />
-  <show_ruler>true</show_ruler>
-  <height>235</height>
-  <name>arrayPlot</name>
-  <snap_to_geometry>false</snap_to_geometry>
-  <show_grid>false</show_grid>
-  <background_color>
-    <color name="Gray_0" red="255" green="255" blue="255" />
-  </background_color>
-  <foreground_color>
-    <color name="Gray_14" red="0" green="0" blue="0" />
-  </foreground_color>
-  <widget_type>Display</widget_type>
-  <show_close_button>true</show_close_button>
-  <width>400</width>
-  <rules />
-  <show_edit_range>true</show_edit_range>
-  <grid_space>5</grid_space>
-  <auto_scale_widgets>
-    <auto_scale_widgets>false</auto_scale_widgets>
-    <min_width>-1</min_width>
-    <min_height>-1</min_height>
-  </auto_scale_widgets>
-  <actions hook="false" hook_all="false" />
-  <y>571</y>
-  <x>578</x>
-  <widget typeId="org.csstudio.opibuilder.widgets.xyGraph" version="1.0">
-    <trace_1_visible>true</trace_1_visible>
-    <trace_14_concatenate_data>true</trace_14_concatenate_data>
-    <trace_9_y_pv></trace_9_y_pv>
-    <trace_6_update_delay>100</trace_6_update_delay>
-    <trace_14_update_mode>0</trace_14_update_mode>
-    <trace_1_trace_type>0</trace_1_trace_type>
-    <trace_16_buffer_size>100</trace_16_buffer_size>
-    <trace_12_y_axis_index>1</trace_12_y_axis_index>
-    <trace_4_point_style>0</trace_4_point_style>
-    <axis_3_dash_grid_line>true</axis_3_dash_grid_line>
-    <trace_17_x_pv_value />
-    <trace_0_trace_color>
-      <color red="0" green="0" blue="0" />
-    </trace_0_trace_color>
-    <trace_18_point_style>0</trace_18_point_style>
-    <trace_19_buffer_size>100</trace_19_buffer_size>
-    <trace_10_point_style>0</trace_10_point_style>
-    <axis_0_auto_scale>true</axis_0_auto_scale>
-    <trace_3_trace_type>0</trace_3_trace_type>
-    <axis_3_minimum>0.0</axis_3_minimum>
-    <trace_18_y_pv></trace_18_y_pv>
-    <trace_18_anti_alias>true</trace_18_anti_alias>
-    <trace_14_y_pv_value />
-    <trace_9_buffer_size>100</trace_9_buffer_size>
-    <trace_4_update_delay>100</trace_4_update_delay>
-    <axis_1_time_format>0</axis_1_time_format>
-    <trace_19_update_delay>100</trace_19_update_delay>
-    <trace_11_point_style>0</trace_11_point_style>
-    <trace_1_point_size>4</trace_1_point_size>
-    <trace_10_y_axis_index>1</trace_10_y_axis_index>
-    <trace_0_plot_mode>0</trace_0_plot_mode>
-    <trace_15_trace_type>0</trace_15_trace_type>
-    <trace_13_update_mode>0</trace_13_update_mode>
-    <trace_18_buffer_size>100</trace_18_buffer_size>
-    <trace_count>1</trace_count>
-    <trace_4_concatenate_data>true</trace_4_concatenate_data>
-    <trace_2_y_pv_value />
-    <trace_14_visible>true</trace_14_visible>
-    <trace_13_x_pv></trace_13_x_pv>
-    <axis_0_show_grid>true</axis_0_show_grid>
-    <axis_2_log_scale>false</axis_2_log_scale>
-    <trace_18_concatenate_data>true</trace_18_concatenate_data>
-    <trace_16_x_pv_value />
-    <trace_11_line_width>1</trace_11_line_width>
-    <axis_0_log_scale>false</axis_0_log_scale>
-    <trace_11_x_axis_index>0</trace_11_x_axis_index>
-    <trace_3_point_size>4</trace_3_point_size>
-    <trace_19_name>$(trace_19_y_pv)</trace_19_name>
-    <trace_9_visible>true</trace_9_visible>
-    <trace_10_concatenate_data>true</trace_10_concatenate_data>
-    <trace_11_update_mode>0</trace_11_update_mode>
-    <trace_10_line_width>1</trace_10_line_width>
-    <trace_18_x_axis_index>0</trace_18_x_axis_index>
-    <trace_6_trace_type>0</trace_6_trace_type>
-    <trace_3_y_axis_index>1</trace_3_y_axis_index>
-    <trace_7_point_style>0</trace_7_point_style>
-    <trace_1_name>$(trace_1_y_pv)</trace_1_name>
-    <trace_17_name>$(trace_17_y_pv)</trace_17_name>
-    <trace_12_concatenate_data>true</trace_12_concatenate_data>
-    <axis_2_grid_color>
-      <color red="200" green="200" blue="200" />
-    </axis_2_grid_color>
-    <trace_12_point_style>0</trace_12_point_style>
-    <axis_2_minimum>0.0</axis_2_minimum>
-    <trace_4_line_width>1</trace_4_line_width>
-    <trace_0_point_size>4</trace_0_point_size>
-    <trace_14_x_pv_value />
-    <axis_3_show_grid>true</axis_3_show_grid>
-    <trace_14_point_style>0</trace_14_point_style>
-    <trace_13_point_style>0</trace_13_point_style>
-    <show_legend>true</show_legend>
-    <trace_15_x_axis_index>0</trace_15_x_axis_index>
-    <trace_8_line_width>1</trace_8_line_width>
-    <trace_10_trace_color>
-      <color red="114" green="40" blue="3" />
-    </trace_10_trace_color>
-    <y>15</y>
-    <trace_5_update_mode>0</trace_5_update_mode>
-    <x>0</x>
-    <axis_3_axis_color>
-      <color red="0" green="0" blue="0" />
-    </axis_3_axis_color>
-    <visible>true</visible>
-    <trace_0_x_pv_value />
-    <trace_11_x_pv></trace_11_x_pv>
-    <axis_1_visible>true</axis_1_visible>
-    <trace_13_y_pv_value />
-    <trace_9_x_pv_value />
-    <height>200</height>
-    <axis_1_maximum>100.0</axis_1_maximum>
-    <trace_10_name>$(trace_10_y_pv)</trace_10_name>
-    <trace_3_x_pv_value />
-    <trace_17_trace_color>
-      <color red="255" green="170" blue="0" />
-    </trace_17_trace_color>
-    <trace_9_name>$(trace_9_y_pv)</trace_9_name>
-    <trace_8_buffer_size>100</trace_8_buffer_size>
-    <trace_16_plot_mode>0</trace_16_plot_mode>
-    <trace_9_plot_mode>0</trace_9_plot_mode>
-    <trace_17_anti_alias>true</trace_17_anti_alias>
-    <trace_12_line_width>1</trace_12_line_width>
-    <trace_4_x_pv></trace_4_x_pv>
-    <trace_9_trace_color>
-      <color red="0" green="214" blue="255" />
-    </trace_9_trace_color>
-    <trace_2_x_pv></trace_2_x_pv>
-    <trace_7_update_delay>100</trace_7_update_delay>
-    <axis_1_title_font>
-      <fontdata fontName="Arial" height="9" style="1" />
-    </axis_1_title_font>
-    <border_width>1</border_width>
-    <axis_2_left_bottom_side>true</axis_2_left_bottom_side>
-    <trace_2_x_axis_index>0</trace_2_x_axis_index>
-    <axis_2_show_grid>true</axis_2_show_grid>
-    <trace_3_name>$(trace_3_y_pv)</trace_3_name>
-    <trace_19_plot_mode>0</trace_19_plot_mode>
-    <trace_19_y_pv_value />
-    <axis_0_axis_color>
-      <color red="0" green="0" blue="0" />
-    </axis_0_axis_color>
-    <trace_13_trace_type>0</trace_13_trace_type>
-    <trace_14_trace_type>0</trace_14_trace_type>
-    <axis_1_scale_format></axis_1_scale_format>
-    <trace_13_buffer_size>100</trace_13_buffer_size>
-    <name>XY Graph</name>
-    <trace_11_trace_type>0</trace_11_trace_type>
-    <axis_1_auto_scale_threshold>0.0</axis_1_auto_scale_threshold>
-    <trace_19_concatenate_data>true</trace_19_concatenate_data>
-    <trace_4_buffer_size>100</trace_4_buffer_size>
-    <trace_10_y_pv_value />
-    <trace_0_line_width>1</trace_0_line_width>
-    <trace_16_anti_alias>true</trace_16_anti_alias>
-    <trace_19_line_width>1</trace_19_line_width>
-    <trace_19_x_axis_index>0</trace_19_x_axis_index>
-    <trace_12_x_axis_index>0</trace_12_x_axis_index>
-    <trace_11_plot_mode>0</trace_11_plot_mode>
-    <trace_2_x_pv_value />
-    <axis_3_auto_scale>true</axis_3_auto_scale>
-    <show_toolbar>true</show_toolbar>
-    <trace_13_x_pv_value />
-    <trace_15_buffer_size>100</trace_15_buffer_size>
-    <trace_1_update_delay>100</trace_1_update_delay>
-    <trace_4_x_axis_index>0</trace_4_x_axis_index>
-    <axis_1_show_grid>true</axis_1_show_grid>
-    <trace_9_x_axis_index>0</trace_9_x_axis_index>
-    <trace_15_name>$(trace_15_y_pv)</trace_15_name>
-    <axis_0_grid_color>
-      <color red="200" green="200" blue="200" />
-    </axis_0_grid_color>
-    <trace_5_plot_mode>0</trace_5_plot_mode>
-    <trace_17_update_delay>100</trace_17_update_delay>
-    <trace_19_anti_alias>true</trace_19_anti_alias>
-    <trace_2_line_width>1</trace_2_line_width>
-    <trace_19_x_pv_value />
-    <trace_1_line_width>1</trace_1_line_width>
-    <trace_18_plot_mode>0</trace_18_plot_mode>
-    <trace_2_y_axis_index>1</trace_2_y_axis_index>
-    <axis_0_auto_scale_threshold>0.0</axis_0_auto_scale_threshold>
-    <trace_15_point_style>0</trace_15_point_style>
-    <trace_2_trace_type>0</trace_2_trace_type>
-    <trace_0_name>$(trace_0_y_pv)</trace_0_name>
-    <trace_1_anti_alias>true</trace_1_anti_alias>
-    <axis_3_auto_scale_threshold>0.0</axis_3_auto_scale_threshold>
-    <trace_3_x_pv></trace_3_x_pv>
-    <trace_11_y_axis_index>1</trace_11_y_axis_index>
-    <trace_13_point_size>4</trace_13_point_size>
-    <axis_2_maximum>100.0</axis_2_maximum>
-    <trace_0_update_mode>0</trace_0_update_mode>
-    <trace_19_visible>true</trace_19_visible>
-    <trace_15_concatenate_data>true</trace_15_concatenate_data>
-    <trace_17_y_axis_index>1</trace_17_y_axis_index>
-    <trace_10_y_pv></trace_10_y_pv>
-    <trace_9_x_pv></trace_9_x_pv>
-    <trace_1_buffer_size>100</trace_1_buffer_size>
-    <axis_0_axis_title></axis_0_axis_title>
-    <trace_3_line_width>1</trace_3_line_width>
-    <trace_11_trace_color>
-      <color red="219" green="128" blue="4" />
-    </trace_11_trace_color>
-    <trace_14_name>$(trace_14_y_pv)</trace_14_name>
-    <wuid>39872d87:13e14015f61:7ec5</wuid>
-    <trace_2_buffer_size>100</trace_2_buffer_size>
-    <trace_18_trace_color>
-      <color red="255" green="0" blue="240" />
-    </trace_18_trace_color>
-    <axis_1_axis_color>
-      <color red="0" green="0" blue="0" />
-    </axis_1_axis_color>
-    <trace_19_point_size>4</trace_19_point_size>
-    <trace_2_update_mode>0</trace_2_update_mode>
-    <foreground_color>
-      <color red="0" green="0" blue="255" />
-    </foreground_color>
-    <trace_15_plot_mode>0</trace_15_plot_mode>
-    <trace_1_update_mode>0</trace_1_update_mode>
-    <backcolor_alarm_sensitive>false</backcolor_alarm_sensitive>
-    <axis_1_axis_title>$(YLABEL)</axis_1_axis_title>
-    <trace_4_anti_alias>true</trace_4_anti_alias>
-    <trace_12_x_pv_value />
-    <trace_9_line_width>1</trace_9_line_width>
-    <trace_7_x_pv_value />
-    <trace_4_plot_mode>0</trace_4_plot_mode>
-    <axis_2_y_axis>true</axis_2_y_axis>
-    <trace_3_update_mode>0</trace_3_update_mode>
-    <axis_0_maximum>100.0</axis_0_maximum>
-    <trace_14_update_delay>100</trace_14_update_delay>
-    <trace_7_anti_alias>true</trace_7_anti_alias>
-    <trace_9_point_size>4</trace_9_point_size>
-    <trace_12_update_delay>100</trace_12_update_delay>
-    <trace_19_trace_type>0</trace_19_trace_type>
-    <trace_16_name>$(trace_16_y_pv)</trace_16_name>
-    <axis_0_scale_font>
-      <opifont.name fontName="Sans" height="10" style="0">Default</opifont.name>
-    </axis_0_scale_font>
-    <trace_19_y_axis_index>1</trace_19_y_axis_index>
-    <trace_17_trace_type>0</trace_17_trace_type>
-    <trace_12_visible>true</trace_12_visible>
-    <trace_12_name>$(trace_12_y_pv)</trace_12_name>
-    <trace_2_point_size>4</trace_2_point_size>
-    <trace_12_plot_mode>0</trace_12_plot_mode>
-    <trace_16_trace_type>0</trace_16_trace_type>
-    <trace_4_y_axis_index>1</trace_4_y_axis_index>
-    <trace_12_update_mode>0</trace_12_update_mode>
-    <trace_6_plot_mode>0</trace_6_plot_mode>
-    <trace_10_x_axis_index>0</trace_10_x_axis_index>
-    <trace_1_y_pv_value />
-    <axis_0_time_format>0</axis_0_time_format>
-    <trace_8_name>$(trace_8_y_pv)</trace_8_name>
-    <trace_4_update_mode>0</trace_4_update_mode>
-    <trace_14_x_pv></trace_14_x_pv>
-    <trace_15_y_pv_value />
-    <trace_8_point_style>0</trace_8_point_style>
-    <trace_10_x_pv></trace_10_x_pv>
-    <trace_10_visible>true</trace_10_visible>
-    <trace_3_concatenate_data>true</trace_3_concatenate_data>
-    <trace_3_trace_color>
-      <color red="0" green="0" blue="0" />
-    </trace_3_trace_color>
-    <trace_15_y_pv></trace_15_y_pv>
-    <trace_11_y_pv_value />
-    <trace_1_plot_mode>0</trace_1_plot_mode>
-    <trace_8_visible>true</trace_8_visible>
-    <trace_8_x_pv_value />
-    <trace_7_visible>true</trace_7_visible>
-    <trace_5_name>$(trace_5_y_pv)</trace_5_name>
-    <trace_17_concatenate_data>true</trace_17_concatenate_data>
-    <trace_10_buffer_size>100</trace_10_buffer_size>
-    <trace_7_y_axis_index>1</trace_7_y_axis_index>
-    <trace_6_point_style>0</trace_6_point_style>
-    <trace_13_x_axis_index>0</trace_13_x_axis_index>
-    <axis_1_log_scale>false</axis_1_log_scale>
-    <trace_16_x_pv></trace_16_x_pv>
-    <trace_17_line_width>1</trace_17_line_width>
-    <trace_8_trace_type>0</trace_8_trace_type>
-    <trace_15_x_pv_value />
-    <trace_18_y_pv_value />
-    <trace_1_point_style>0</trace_1_point_style>
-    <trace_5_x_pv></trace_5_x_pv>
-    <trace_18_y_axis_index>1</trace_18_y_axis_index>
-    <axis_2_auto_scale_threshold>0.0</axis_2_auto_scale_threshold>
-    <trace_18_x_pv_value />
-    <trace_13_update_delay>100</trace_13_update_delay>
-    <trace_7_y_pv></trace_7_y_pv>
-    <trace_13_name>$(trace_13_y_pv)</trace_13_name>
-    <trace_11_x_pv_value />
-    <trace_11_anti_alias>true</trace_11_anti_alias>
-    <trace_5_point_style>0</trace_5_point_style>
-    <axis_3_maximum>100.0</axis_3_maximum>
-    <trace_2_plot_mode>0</trace_2_plot_mode>
-    <title_font>
-      <fontdata fontName="Arial" height="12" style="1" />
-    </title_font>
-    <trace_6_y_pv></trace_6_y_pv>
-    <trace_8_y_pv_value />
-    <trace_8_trace_color>
-      <color red="0" green="255" blue="11" />
-    </trace_8_trace_color>
-    <trace_3_buffer_size>100</trace_3_buffer_size>
-    <axis_1_dash_grid_line>true</axis_1_dash_grid_line>
-    <trace_12_buffer_size>100</trace_12_buffer_size>
-    <trace_10_update_mode>0</trace_10_update_mode>
-    <trace_13_anti_alias>true</trace_13_anti_alias>
-    <trace_16_visible>true</trace_16_visible>
-    <axis_2_dash_grid_line>true</axis_2_dash_grid_line>
-    <trace_15_anti_alias>true</trace_15_anti_alias>
-    <trace_15_update_mode>0</trace_15_update_mode>
-    <trace_8_update_mode>0</trace_8_update_mode>
-    <axis_0_visible>true</axis_0_visible>
-    <trace_14_plot_mode>0</trace_14_plot_mode>
-    <trace_10_plot_mode>0</trace_10_plot_mode>
-    <trace_14_y_pv></trace_14_y_pv>
-    <trace_19_y_pv></trace_19_y_pv>
-    <trace_6_point_size>4</trace_6_point_size>
-    <trace_14_trace_color>
-      <color red="33" green="179" blue="33" />
-    </trace_14_trace_color>
-    <forecolor_alarm_sensitive>false</forecolor_alarm_sensitive>
-    <trace_18_trace_type>0</trace_18_trace_type>
-    <trace_7_plot_mode>0</trace_7_plot_mode>
-    <trace_12_x_pv></trace_12_x_pv>
-    <trace_1_x_pv_value />
-    <trace_14_y_axis_index>1</trace_14_y_axis_index>
-    <trace_6_concatenate_data>true</trace_6_concatenate_data>
-    <axis_0_scale_format></axis_0_scale_format>
-    <trace_4_name>$(trace_4_y_pv)</trace_4_name>
-    <trace_15_line_width>1</trace_15_line_width>
-    <trace_11_concatenate_data>true</trace_11_concatenate_data>
-    <trace_6_visible>true</trace_6_visible>
-    <trace_7_buffer_size>100</trace_7_buffer_size>
-    <trace_12_trace_type>0</trace_12_trace_type>
-    <trace_4_trace_type>0</trace_4_trace_type>
-    <trace_0_x_pv>$(X)</trace_0_x_pv>
-    <trace_16_update_delay>100</trace_16_update_delay>
-    <trace_16_update_mode>0</trace_16_update_mode>
-    <trace_13_visible>true</trace_13_visible>
-    <trace_7_line_width>1</trace_7_line_width>
-    <trace_0_y_pv>$(Y)</trace_0_y_pv>
-    <trace_18_name>$(trace_18_y_pv)</trace_18_name>
-    <trace_12_y_pv></trace_12_y_pv>
-    <axis_3_visible>true</axis_3_visible>
-    <trace_19_x_pv></trace_19_x_pv>
-    <trace_5_point_size>4</trace_5_point_size>
-    <trace_18_x_pv></trace_18_x_pv>
-    <trace_18_point_size>4</trace_18_point_size>
-    <trace_6_x_pv_value />
-    <trace_6_update_mode>0</trace_6_update_mode>
-    <axis_0_title_font>
-      <fontdata fontName="Arial" height="9" style="1" />
-    </axis_0_title_font>
-    <trace_5_buffer_size>100</trace_5_buffer_size>
-    <trace_16_y_pv_value />
-    <trace_3_point_style>0</trace_3_point_style>
-    <axis_1_auto_scale>true</axis_1_auto_scale>
-    <trace_7_x_axis_index>0</trace_7_x_axis_index>
-    <trace_2_update_delay>100</trace_2_update_delay>
-    <trace_16_line_width>1</trace_16_line_width>
-    <trace_4_point_size>4</trace_4_point_size>
-    <trace_0_concatenate_data>true</trace_0_concatenate_data>
-    <trace_9_update_delay>100</trace_9_update_delay>
-    <trace_5_y_axis_index>1</trace_5_y_axis_index>
-    <trace_7_update_mode>0</trace_7_update_mode>
-    <trace_9_trace_type>0</trace_9_trace_type>
-    <trace_3_y_pv></trace_3_y_pv>
-    <actions hook="false" hook_all="false" />
-    <trigger_pv_value />
-    <trace_13_concatenate_data>true</trace_13_concatenate_data>
-    <trace_1_x_pv></trace_1_x_pv>
-    <trace_14_anti_alias>true</trace_14_anti_alias>
-    <show_plot_area_border>false</show_plot_area_border>
-    <trace_4_trace_color>
-      <color red="128" green="0" blue="255" />
-    </trace_4_trace_color>
-    <axis_2_axis_title>Secondary Axis (2)</axis_2_axis_title>
-    <trace_11_y_pv></trace_11_y_pv>
-    <trace_15_visible>true</trace_15_visible>
-    <trace_8_x_axis_index>0</trace_8_x_axis_index>
-    <widget_type>XY Graph</widget_type>
-    <enabled>true</enabled>
-    <width>400</width>
-    <trace_17_buffer_size>100</trace_17_buffer_size>
-    <trace_14_x_axis_index>0</trace_14_x_axis_index>
-    <axis_3_title_font>
-      <fontdata fontName="Arial" height="9" style="1" />
-    </axis_3_title_font>
-    <trace_13_y_axis_index>1</trace_13_y_axis_index>
-    <trace_16_x_axis_index>0</trace_16_x_axis_index>
-    <trace_0_buffer_size>100</trace_0_buffer_size>
-    <trace_11_update_delay>100</trace_11_update_delay>
-    <trigger_pv></trigger_pv>
-    <trace_19_point_style>0</trace_19_point_style>
-    <trace_6_trace_color>
-      <color red="255" green="0" blue="240" />
-    </trace_6_trace_color>
-    <trace_5_trace_type>0</trace_5_trace_type>
-    <trace_15_y_axis_index>1</trace_15_y_axis_index>
-    <trace_14_buffer_size>100</trace_14_buffer_size>
-    <axis_3_grid_color>
-      <color red="200" green="200" blue="200" />
-    </axis_3_grid_color>
-    <axis_count>2</axis_count>
-    <trace_2_name>$(trace_2_y_pv)</trace_2_name>
-    <trace_4_visible>true</trace_4_visible>
-    <trace_10_point_size>4</trace_10_point_size>
-    <trace_5_concatenate_data>true</trace_5_concatenate_data>
-    <transparent>false</transparent>
-    <trace_2_anti_alias>true</trace_2_anti_alias>
-    <trace_7_trace_type>0</trace_7_trace_type>
-    <axis_0_dash_grid_line>true</axis_0_dash_grid_line>
-    <trace_2_visible>true</trace_2_visible>
-    <trace_13_y_pv></trace_13_y_pv>
-    <trace_5_update_delay>100</trace_5_update_delay>
-    <trace_8_y_pv></trace_8_y_pv>
-    <trace_0_x_axis_index>0</trace_0_x_axis_index>
-    <trace_17_point_style>0</trace_17_point_style>
-    <trace_12_anti_alias>true</trace_12_anti_alias>
-    <trace_18_update_delay>100</trace_18_update_delay>
-    <trace_16_trace_color>
-      <color red="128" green="0" blue="255" />
-    </trace_16_trace_color>
-    <axis_2_time_format>0</axis_2_time_format>
-    <trace_6_y_axis_index>1</trace_6_y_axis_index>
-    <trace_17_point_size>4</trace_17_point_size>
-    <trace_6_y_pv_value />
-    <trace_0_point_style>0</trace_0_point_style>
-    <tooltip>$(trace_0_y_pv)
-$(trace_0_y_pv_value)</tooltip>
-    <trace_11_name>$(trace_11_y_pv)</trace_11_name>
-    <border_alarm_sensitive>true</border_alarm_sensitive>
-    <axis_2_title_font>
-      <fontdata fontName="Arial" height="9" style="1" />
-    </axis_2_title_font>
-    <trace_4_y_pv></trace_4_y_pv>
-    <trace_12_point_size>4</trace_12_point_size>
-    <trace_1_x_axis_index>0</trace_1_x_axis_index>
-    <trace_11_buffer_size>100</trace_11_buffer_size>
-    <trace_6_name>$(trace_6_y_pv)</trace_6_name>
-    <trace_8_point_size>4</trace_8_point_size>
-    <trace_19_update_mode>0</trace_19_update_mode>
-    <trace_8_y_axis_index>1</trace_8_y_axis_index>
-    <trace_16_concatenate_data>true</trace_16_concatenate_data>
-    <trace_9_point_style>0</trace_9_point_style>
-    <trace_0_y_axis_index>1</trace_0_y_axis_index>
-    <trace_3_visible>true</trace_3_visible>
-    <axis_1_scale_font>
-      <opifont.name fontName="Sans" height="10" style="0">Default</opifont.name>
-    </axis_1_scale_font>
-    <trace_17_update_mode>0</trace_17_update_mode>
-    <border_style>0</border_style>
-    <trace_13_line_width>1</trace_13_line_width>
-    <trace_3_x_axis_index>0</trace_3_x_axis_index>
-    <plot_area_background_color>
-      <color red="255" green="255" blue="255" />
-    </plot_area_background_color>
-    <trace_2_concatenate_data>true</trace_2_concatenate_data>
-    <trace_10_trace_type>0</trace_10_trace_type>
-    <axis_3_scale_font>
-      <opifont.name fontName="Sans" height="10" style="0">Default</opifont.name>
-    </axis_3_scale_font>
-    <trace_16_point_size>4</trace_16_point_size>
-    <trace_15_x_pv></trace_15_x_pv>
-    <trace_2_point_style>0</trace_2_point_style>
-    <axis_3_y_axis>true</axis_3_y_axis>
-    <trace_9_update_mode>0</trace_9_update_mode>
-    <title></title>
-    <axis_3_axis_title>Secondary Axis (3)</axis_3_axis_title>
-    <axis_3_log_scale>false</axis_3_log_scale>
-    <trace_8_anti_alias>true</trace_8_anti_alias>
-    <trace_13_plot_mode>0</trace_13_plot_mode>
-    <trace_10_update_delay>100</trace_10_update_delay>
-    <trace_1_y_pv></trace_1_y_pv>
-    <trace_8_x_pv></trace_8_x_pv>
-    <axis_2_auto_scale>true</axis_2_auto_scale>
-    <trace_17_plot_mode>0</trace_17_plot_mode>
-    <pv_name></pv_name>
-    <axis_1_minimum>0.0</axis_1_minimum>
-    <axis_2_axis_color>
-      <color red="0" green="0" blue="0" />
-    </axis_2_axis_color>
-    <trace_3_plot_mode>0</trace_3_plot_mode>
-    <trace_14_point_size>4</trace_14_point_size>
-    <trace_12_y_pv_value />
-    <trace_0_visible>true</trace_0_visible>
-    <trace_6_line_width>1</trace_6_line_width>
-    <axis_2_scale_font>
-      <opifont.name fontName="Sans" height="10" style="0">Default</opifont.name>
-    </axis_2_scale_font>
-    <trace_11_visible>true</trace_11_visible>
-    <axis_2_scale_format></axis_2_scale_format>
-    <trace_7_x_pv></trace_7_x_pv>
-    <axis_0_minimum>0.0</axis_0_minimum>
-    <trace_5_trace_color>
-      <color red="255" green="170" blue="0" />
-    </trace_5_trace_color>
-    <border_color>
-      <color red="0" green="128" blue="255" />
-    </border_color>
-    <trace_7_trace_color>
-      <color red="243" green="132" blue="132" />
-    </trace_7_trace_color>
-    <trace_16_y_pv></trace_16_y_pv>
-    <trace_8_update_delay>100</trace_8_update_delay>
-    <trace_11_point_size>4</trace_11_point_size>
-    <axis_3_left_bottom_side>true</axis_3_left_bottom_side>
-    <trace_3_update_delay>100</trace_3_update_delay>
-    <trace_9_concatenate_data>true</trace_9_concatenate_data>
-    <trace_9_y_axis_index>1</trace_9_y_axis_index>
-    <trace_8_concatenate_data>true</trace_8_concatenate_data>
-    <scripts />
-    <trace_5_visible>true</trace_5_visible>
-    <trace_0_anti_alias>true</trace_0_anti_alias>
-    <trace_4_x_pv_value />
-    <axis_3_time_format>0</axis_3_time_format>
-    <trace_13_trace_color>
-      <color red="242" green="26" blue="26" />
-    </trace_13_trace_color>
-    <trace_6_x_axis_index>0</trace_6_x_axis_index>
-    <trace_16_y_axis_index>1</trace_16_y_axis_index>
-    <trace_12_trace_color>
-      <color red="21" green="21" blue="196" />
-    </trace_12_trace_color>
-    <background_color>
-      <color red="240" green="240" blue="240" />
-    </background_color>
-    <trace_10_anti_alias>true</trace_10_anti_alias>
-    <trace_6_buffer_size>100</trace_6_buffer_size>
-    <trace_1_trace_color>
-      <color red="242" green="26" blue="26" />
-    </trace_1_trace_color>
-    <trace_2_trace_color>
-      <color red="33" green="179" blue="33" />
-    </trace_2_trace_color>
-    <trace_9_y_pv_value />
-    <trace_7_point_size>4</trace_7_point_size>
-    <trace_19_trace_color>
-      <color red="243" green="132" blue="132" />
-    </trace_19_trace_color>
-    <trace_6_anti_alias>true</trace_6_anti_alias>
-    <trace_17_x_axis_index>0</trace_17_x_axis_index>
-    <trace_5_y_pv_value />
-    <pv_value />
-    <trace_17_y_pv></trace_17_y_pv>
-    <trace_10_x_pv_value />
-    <trace_5_x_pv_value />
-    <trace_17_y_pv_value />
-    <trace_5_line_width>1</trace_5_line_width>
-    <trace_4_y_pv_value />
-    <trace_8_plot_mode>0</trace_8_plot_mode>
-    <trace_18_visible>true</trace_18_visible>
-    <trace_5_y_pv></trace_5_y_pv>
-    <trace_16_point_style>0</trace_16_point_style>
-    <trace_14_line_width>1</trace_14_line_width>
-    <trace_7_y_pv_value />
-    <trace_5_x_axis_index>0</trace_5_x_axis_index>
-    <axis_2_visible>true</axis_2_visible>
-    <trace_1_y_axis_index>1</trace_1_y_axis_index>
-    <trace_3_anti_alias>true</trace_3_anti_alias>
-    <trace_5_anti_alias>true</trace_5_anti_alias>
-    <axis_3_scale_format></axis_3_scale_format>
-    <scale_options>
-      <width_scalable>true</width_scalable>
-      <height_scalable>true</height_scalable>
-      <keep_wh_ratio>false</keep_wh_ratio>
-    </scale_options>
-    <trace_17_x_pv></trace_17_x_pv>
-    <trace_1_concatenate_data>true</trace_1_concatenate_data>
-    <trace_6_x_pv></trace_6_x_pv>
-    <trace_15_update_delay>100</trace_15_update_delay>
-    <trace_18_line_width>1</trace_18_line_width>
-    <trace_18_update_mode>0</trace_18_update_mode>
-    <rules />
-    <trace_2_y_pv></trace_2_y_pv>
-    <trace_0_y_pv_value />
-    <trace_0_trace_type>0</trace_0_trace_type>
-    <axis_1_grid_color>
-      <color red="200" green="200" blue="200" />
-    </axis_1_grid_color>
-    <trace_3_y_pv_value />
-    <trace_7_concatenate_data>true</trace_7_concatenate_data>
-    <trace_0_update_delay>100</trace_0_update_delay>
-    <trace_15_trace_color>
-      <color red="0" green="0" blue="0" />
-    </trace_15_trace_color>
-    <trace_9_anti_alias>true</trace_9_anti_alias>
-    <trace_15_point_size>4</trace_15_point_size>
-    <trace_17_visible>true</trace_17_visible>
-    <trace_7_name>$(trace_7_y_pv)</trace_7_name>
-  </widget>
-  <widget typeId="org.csstudio.opibuilder.widgets.Label" version="1.0">
-    <visible>true</visible>
-    <vertical_alignment>1</vertical_alignment>
-    <wuid>39872d87:13e14015f61:7ec6</wuid>
-    <auto_size>false</auto_size>
-    <scripts />
-    <height>13</height>
-    <name>Label</name>
-    <scale_options>
-      <width_scalable>true</width_scalable>
-      <height_scalable>true</height_scalable>
-      <keep_wh_ratio>false</keep_wh_ratio>
-    </scale_options>
-    <transparent>true</transparent>
-    <show_scrollbar>false</show_scrollbar>
-    <background_color>
-      <color red="255" green="255" blue="255" />
-    </background_color>
-    <foreground_color>
-      <color name="Gray_14" red="0" green="0" blue="0" />
-    </foreground_color>
-    <widget_type>Label</widget_type>
-    <enabled>true</enabled>
-    <text>Y=$(Y)</text>
-    <font>
-      <fontdata fontName="Sans" height="8" style="0" />
-    </font>
-    <width>400</width>
-    <border_style>0</border_style>
-    <rules />
-    <border_width>1</border_width>
-    <border_color>
-      <color red="0" green="128" blue="255" />
-    </border_color>
-    <horizontal_alignment>1</horizontal_alignment>
-    <actions hook="false" hook_all="false" />
-    <y>0</y>
-    <wrap_words>true</wrap_words>
-    <tooltip></tooltip>
-    <x>0</x>
-  </widget>
-  <widget typeId="org.csstudio.opibuilder.widgets.Label" version="1.0">
-    <visible>true</visible>
-    <vertical_alignment>1</vertical_alignment>
-    <wuid>39872d87:13e14015f61:7ec7</wuid>
-    <auto_size>false</auto_size>
-    <scripts />
-    <height>13</height>
-    <name>Label</name>
-    <scale_options>
-      <width_scalable>true</width_scalable>
-      <height_scalable>true</height_scalable>
-      <keep_wh_ratio>false</keep_wh_ratio>
-    </scale_options>
-    <transparent>true</transparent>
-    <show_scrollbar>false</show_scrollbar>
-    <background_color>
-      <color red="255" green="255" blue="255" />
-    </background_color>
-    <foreground_color>
-      <color name="Gray_14" red="0" green="0" blue="0" />
-    </foreground_color>
-    <widget_type>Label</widget_type>
-    <enabled>true</enabled>
-    <text>X=$(X)</text>
-    <font>
-      <fontdata fontName="Sans" height="8" style="0" />
-    </font>
-    <width>400</width>
-    <border_style>0</border_style>
-    <rules />
-    <border_width>1</border_width>
-    <border_color>
-      <color red="0" green="128" blue="255" />
-    </border_color>
-    <horizontal_alignment>1</horizontal_alignment>
-    <actions hook="false" hook_all="false" />
-    <y>218</y>
-    <wrap_words>true</wrap_words>
-    <tooltip></tooltip>
-    <x>0</x>
-  </widget>
-</display>
\ No newline at end of file
